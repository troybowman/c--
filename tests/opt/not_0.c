../real/not_0.c