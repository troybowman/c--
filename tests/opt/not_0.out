this should print
this should also print
finally, this should print
